### PR TITLE
Remove spurious print statement

### PR DESCRIPTION
**Pull request by bruth**
This is causing JSON-rendered output to be invalid.

Signed-off-by: Byron Ruth <byron@nats.io>

### CODE REVIEW

**Comment by bruth** [comment]
The failure appears to be an install while installing staticcheck.. although I was able to install the same version locally, so I am not sure why it is failing in CI.

**Comment by ripienaar** [comment]
Yeah unfortunately staticcheck has stopped supporting go 1.18, you can remove 1.18 from the github action and it wil go away

**Comment by bruth** [comment]
Does an older version of staticcheck support Go 1.18? I suppose we could use that version for that specific build?

**Comment by ripienaar** [comment]
Happy to drop go 1.18, its not supported by go team anymore, dropping it is easiest :)

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go: [ 1.18, 1.19 ]
+        go: [ 1.19, 1.20 ]
```
Comment (ripienaar) [inline .github/workflows/test.yaml:8]: you'll have to quote `"1.20"`, easiest to just quote both to avoid the mistake in future (1.20 is 1.2 not 1.20)

Comment (bruth) [reply .github/workflows/test.yaml:8]: :man_facepalming: Yep.

Comment (ripienaar) [reply .github/workflows/test.yaml]: Made the same mistake on other projects hehe

**Comment by ripienaar** [comment]
```
 (220, 2)  SA1019  rand.Seed has been deprecated since Go 1.20 and an alternative has been available since Go 1.0: Programs that call Seed and then expect a specific sequence of results from the global random source (using functions such as Int) can be broken when a dependency changes how much it consumes from the global random source. To avoid such breakages, programs that need a specific result sequence should use NewRand(NewSource(seed)) to obtain a random generator that other packages cannot access. 
```

You can fix it, or just drop 1.20 for now - this is why I suggested not adding 1.20 yet as I know there are a few changes I wanted to go through first

**Review by ripienaar — APPROVED** [review]
LGTM

Ignore the test requirement, I already updated github but it wont relfect on this PR

**Comment by bruth** [comment]
:+1: Sounds good. I cannot merge, so feel free.

**Pull request merged**